src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -92,6 +92,8 @@ public static partial class PlatformDetection
         public static bool IsBrowserDomSupportedOrNotBrowser => IsNotBrowser || IsBrowserDomSupported;
         public static bool IsNotBrowserDomSupported => !IsBrowserDomSupported;
         public static bool IsWebSocketSupported => IsEnvironmentVariableTrue("IsWebSocketSupported");
+        public static bool IsNodeJS => IsEnvironmentVariableTrue("IsNodeJS");
```

Comment:
Just curious - is it possible to detect this without using environment variables?

Reply:
We compute that value in the [`test-main.js`](https://github.com/dotnet/runtime/blob/158f0897958f857d3fe2b142371a3831a7d4f023/src/mono/wasm/test-main.js#L11). Once we have an interop API that is referenceable from `TestUtilities`, we could invoke the same expression here.